### ISSUE TITLE
Logo disappears on smaller screens

### ISSUE THREAD

**Issue by libcce** [carpentries/carpentries.org-archive#67]
Logo disappears on smaller screens, e.g. iPhone:
http://quirktools.com/screenfly/#u=https%3A//carpentries.org/&w=375&h=667&a=37

**Comment by maneesha** [comment]
I don't know enough about design to know whether this is intentional or not. Any suggestions about who might be able to help? @tracykteal @ErinBecker @fmichonneau @serahrono

 It does not make the mobile screen display look bad.  I'll close this unless someone feels strongly it needs to be fixed.

**Comment by rgaiacs** [comment]
> I don't know enough about design to know whether this is intentional or not.

This is a bug. You want to replace the

```
<h1 class="show-for-small-only"><a href="https://carpentries.org"> The Carpentries</a></h1>
```

with

```
<a id="logo" href="https://carpentries.org/" title="The Carpentries – We teach foundational coding and data science skills to researchers worldwide."> <img src="https://carpentries.org/assets/img/TheCarpentries.svg" alt="The Carpentries – We teach foundational coding and data science skills to researchers worldwide."> </a>
```

You need to double check that the image is scale down.

**Comment by maneesha** [comment]
Thanks @rgaiacs ! Would you want to put in a PR?

**Comment by rgaiacs** [comment]
> Would you want to put in a PR?

I would prefer defer to @serahrono or @fmichonneau.

**Comment by fmichonneau** [comment]
I had looked into it when we first set up the website. It's how the template is designed to make it responsive, and changing it requires a lot of modifications. For instance, if we try to squeeze the logo and the text on small screens, then we also need to modify the height of the black and white image.

On small screens, "The Carpentries" (as text and not a logo) appears on the top nav bar making our "brand" clear. I'd put this in the "wontfix" category.
